tests/test262/interpret-results.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
+
+module.exports = function report(results, allowed) {
+  var expected = {
+    success: [],
+    failure: [],
+    typeI: [],
+    typeII: []
+  };
+  var unexpected = {
+    success: [],
+    failure: [],
+    typeI: [],
+    typeII: [],
+    unrecognized: null
+  };
+  var totalUnexpected;
+
+  results.forEach(function(result) {
+    var isAllowed = allowed[result.name];
+    delete allowed[result.name];
+
+    if (!!result.parseFailure === result.expected) {
+      if (isAllowed) {
+        if (result.parseFailure) {
+          unexpected.failure.push(result.name);
+        } else {
+          unexpected.success.push(result.name);
+        }
+      } else {
+        if (result.parseFailure) {
+          expected.failure.push(result.name);
+        } else {
+          expected.success.push(result.name);
+        }
+      }
+    } else {
+      if (isAllowed) {
+        if (result.parseFailure) {
+          expected.typeI.push(result.name);
```

Comment:
Those aren't lazy programmer terms--they come from statistical testing. But
there are plenty of people who aren't familiar with that, so your point is
well-taken. The terms "passes" and "failures" are a little too vague here
because both conditions describe failures. A common alternative is "false
positives" and "false negatives." I probably should have used those from the
start!